src/client/dfs/cont.c

```diff
@@ -83,7 +84,7 @@ dfs_cont_create(daos_handle_t poh, uuid_t *cuuid, dfs_attr_t *attr, daos_handle_
 			dattr.da_chunk_size = DFS_DEFAULT_CHUNK_SIZE;
 
 		if (attr->da_hints[0] != 0) {
-			strncpy(dattr.da_hints, attr->da_hints, DAOS_CONT_HINT_MAX_LEN);
+			memcpy(dattr.da_hints, attr->da_hints, DAOS_CONT_HINT_MAX_LEN - 1);
```

Comment:
.

Reply:
- [ ] Restore, add comments and pragma on `strncpy()` usage

Reply:
> * [x]  Restore, add comments and pragma on `strncpy()` usage

Fixed with commit f38fd44a0